### PR TITLE
task(test): setup travis

### PR DESCRIPTION
**Pull request by gnomeontherun**
## Description

Added a first draft of testing using Travis. Lots of room to improve over time.

### What's included?

- Travis CI configuration for unit tests with karma
- Runs only in latest Firefox

##### Screenshots or link to CodePen/Plunker/JSfiddle

https://travis-ci.org/Teradata/covalent/builds/144336920

### CODE REVIEW

**Comment by kyleledbetter** [comment]
Awesome! Can you include the readme badge in this?

**Comment by gnomeontherun** [comment]
Updated.

**Comment by emoralesb05** [comment]
LGTM!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Teradata Web Platform
 
+[![Build Status](https://travis-ci.org/Teradata/covalent.svg?branch=feature%2Ftravis-ci)](https://travis-ci.org/Teradata/covalent)
```
Comment (emoralesb05) [inline README.md]: Should this be pointing to branch master or develop?

**Pull request merged**